core/src/test/scala/org/broadinstitute/dsde/rawls/jobexec/SubmissionMonitorSpec.scala

```diff
@@ -421,10 +421,10 @@ class SubmissionMonitorSpec(_system: ActorSystem) extends TestKit(_system) with
     val outputExpressions = Map("unbound" -> unboundExprStr, "bound" ->  boundExprStr)
     val execOutputs = Map("unbound" -> Left(unboundAttr), "bound" ->  Left(boundAttr))
 
-    val parsedExpr = OutputExpression.build(boundExprStr.value, boundAttr)
-    parsedExpr shouldBe a [scala.util.Success[BoundOutputExpression]]
```

Comment:
Yeah I know I just added this code.  It's a warning thanks to our old friend Type Erasure